app/backend/app.py

```diff
@@ -273,7 +273,7 @@ def create_app():
         AioHttpClientInstrumentor().instrument()
     app = Quart(__name__)
     app.register_blueprint(bp)
-    app.asgi_app = OpenTelemetryMiddleware(app.asgi_app)
+    app.asgi_app = OpenTelemetryMiddleware(app.asgi_app)  # type: ignore[method-assign]
```

Comment:
I had to ignore this one, as apparently mypy doesn't approve of assigning to a method, but this is the official way of using middleware in Quart (and other frameworks).

Reply:
makes sense